src/api/Visualization/types.ts

```diff
@@ -247,17 +250,18 @@ export interface IDeleteVisualizationFolderParams extends IDataSourceParams {
   id: number;
 }
 
-export type VisualizationTree = Tree<
-  {
-    id: number;
-    title: string;
-    shareCount: number;
-    widgetKey?: string; // defined if the visualization has a widget
-    createdAt: string;
-    updatedAt: string;
-  },
-  'visu'
->;
+export interface VisualizationTreeItem {
+  id: number;
+  title: string;
+  shareCount: number;
+  widgetKey?: string; // defined if the visualization has a widget
+  createdAt: string;
+  updatedAt: string;
+  lastLockedByUser: Pick<User, 'id' | 'username' | 'email'>;
```

Comment:
Same here:

1. The `lastLockedByUser` is not always available, the user can be deleted
    - The correct type is `Pick<User, 'username' | 'email'>`
    - We return "deleted user" when the user is deleted